README.rst

```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://travis-ci.org/epam/OneDrive-L.svg?branch=master)](https://travis-ci.org/epam/OneDrive-L)
```

Comment:
Let's change to rst format:
.. image:: https://travis-ci.org/epam/OneDrive-L.svg?branch=master
    :target: https://travis-ci.org/epam/OneDrive-L